ext/MooncakeDiffEqBaseExt.jl

```diff
@@ -0,0 +1,20 @@
+module MooncakeDiffEqBaseExt
+
+using DiffEqBase, Mooncake
+
+Mooncake.@from_rrule(
```

Comment:
I think you'll need the solution adjoints as well.

Should this all live with DiffEqBase?

Reply:
Thanks for this. What do you mean by the solution adjoints here?

I would be happy to move this over to DiffEqBase if that's what you would prefer -- also happy to have it here though 🤷

Reply:
Yeah I'd prefer it live there so that I have some forced interaction with it and start to learn it / use it a bit.

Reply:
Okay -- I'll look into making a PR.